Wikipedia/Code/EditPreviewViewController.swift

```diff
@@ -8,7 +8,7 @@ protocol EditPreviewViewControllerDelegate: NSObjectProtocol {
 class EditPreviewViewController: UIViewController, Themeable, UITextFieldDelegate, UIScrollViewDelegate, WMFOpenExternalLinkDelegate, WMFPreviewSectionLanguageInfoDelegate, WMFPreviewAnchorTapAlertDelegate {
     var section: MWKSection?
     var wikiText = ""
-    var funnel: EditFunnel?
+    private var funnel: EditFunnel?
```

Comment:
making these private should not have worked... looking at where these get set...

Reply:
i think some edit funnels may have not been wired up previously...